include/scenario/Scenario.h

```diff
@@ -31,23 +31,22 @@ namespace acclimate {
 template<class ModelVariant>
 class Scenario {
   protected:
-    const settings::SettingsNode& settings;
+    settings::SettingsNode settings;
```

Comment:
please do similar to `Output.h`: leave settings (all settings) as a `const &` and include a `scenario_node`